proto/osmosis/concentrated-liquidity/genesis.proto

```diff
@@ -57,7 +57,10 @@ message GenesisState {
 
   repeated Position positions = 3 [ (gogoproto.nullable) = false ];
 
-  uint64 next_position_id = 4
+  uint64 position_lock_id = 4
```

Comment:
This doesn't make sense. The lock id must be defined on the proto-generated `PositionData` and that should be replace  `Position` in `GenesisState`.

Each position may have a lock id associated with it as opposed to some single `position_lock_id` in the entire genesis state